bin/dev

```diff
@@ -6,7 +6,7 @@ if ! gem list foreman -i --silent; then
 fi
 
 # Default to port 3000 if not specified
-export PORT="${PORT:-3000}"
+export PORT="${PORT:-ENV.fetch("PORT") { 3000 }}"
```

Comment:
I don't think we'll need to change this—it already allows for a `PORT` configuration with a default